Architecture_Engine/Query/IsValidForInPlaneDistanceCheck.cs

```diff
@@ -36,6 +36,12 @@ public static partial class Query
 
         public static bool IsValidForInPlaneDistanceCheck(this Opening opening1, Opening opening2, double maxDistance = double.NaN, double distanceTolerance = Tolerance.Distance, double angleTolerance = Tolerance.Angle)
         {
+            // Check if openings aren't nulls
+            if (opening1 == null || opening2 == null)
+            {
+                return true;
```

Comment:
You sure this should return `true`? This indicates there is a point in checking distance between `null` openings 🙈

Btw, I would also add an error saying that it is impossible to verify the query because the opening is `null`.

Reply:
You are right, it should return `false`. I will add an error notification 👍